query_optimizer/ExecutionGenerator.cpp

```diff
@@ -761,6 +762,25 @@ void ExecutionGenerator::convertCreateTable(
     ++aid;
   }
 
+  // if specified, set the physical block type
```

Comment:
For the most part, comments are written with regular English grammar. So, capitalize the start of sentences and end them with periods.